build_tools/circle/build_doc.sh

```diff
@@ -57,7 +57,7 @@ get_build_type() {
         echo QUICK BUILD: no changed filenames for $git_range
         return
     fi
-    changed_examples=$(echo "$filenames" | grep -e ^examples/)
+    changed_examples=$(echo "$filenames" | grep -E "^examples/(.*/)*plot_")
```

Comment:
``grep -e`` seems enough?

Reply:
@adrinjalali Hmm, I tried locally and seems that grep -E won't work, I'm not familiar with -E.

Reply:
the `-E` just gives special meaning to `(` and `)`, works for me. For me it's equivalent to `grep -e "^examples/\(.*/\)*plot_"`

Reply:
Apologies I went through your code too quickly.